docs/components/routes.md

```diff
@@ -73,14 +73,15 @@ end
 
 This will generate `userDetails` field on GraphQL side.
 
-## _query_ and _mutation_
+## _query_ and _mutation_ & _subscription_
 
 in case you want to have non-CRUD controller with custom actions you can define your own `query`/`mutation` actions like this:
```

Comment:
```suggestion
in case you want to have non-CRUD controller with custom actions you can define your own `query`/`mutation`/`subscription` actions like this:
```

:de: